fbgemm_gpu/README.md

```diff
@@ -156,6 +156,87 @@ parts of FBGEMM_GPU please see our Wiki (work in progress).
 We have extensively used comments in our source files. The best and up-do-date
 documentation is available in the source files.
 
+# Building API Documentation
```

Comment:
We can put this as a separate README.md in fbgemm_gpu/docs folder, and then link to that file from this README.md (add a ## subsection on docs and link to there)?

Reply:
moved to docs/README.md